definitions/pipelines/immuno.cwl

```diff
@@ -288,11 +288,14 @@ inputs:
     reference_dict:
         type: File
 
-    clinical_hla_alleles:
+    clinical_MHCcI_alleles:
         type: string[]?
-        label: "clinical_calls: Clinical HLA typing results; element format: HLA-X*01:02[/HLA-X...]"
-        doc: |
-          clinical_calls is used to provide clinical HLA typing results in the format HLA-X*01:02[/HLA-X...] when available.
+        label: "Clinical HLA typing results, limited to MHC Class I alleles; element format: HLA-X*01:02[/HLA-X...]"
+        doc: "used to provide clinical HLA typing results in the format HLA-X*01:02[/HLA-X...] when available."
+    clinical_MHCcII_alleles:
+        type: string[]?
+        label: "Clinical HLA typing results, limited to MHC Class II alleles"
+        doc: "clinical_calls is used to provide clinical HLA typing results; separated from class I due to nomenclature inconsistencies"
```

Comment:
I'm guessing this used to be called `clinical_calls` once upon a time? Could probably just take that part out.
```suggestion
        doc: "used to provide clinical HLA typing results; separated from class I due to nomenclature inconsistencies"
```